crates/number/src/conversions.rs

```diff
@@ -80,6 +81,57 @@ pub fn big_decimal_to_big_rational(value: &BigDecimal) -> BigRational {
     BigRational::new(adjusted_numer, denom)
 }
 
+/// Floats in Rust have wrong bytes representation which leaks into
+/// `BigRational` instance when converting from float.
+///
+/// This function converts a decimal string (e.g., `"0.1"`) to an exact
+/// `BigRational`.
+pub fn big_rational_from_decimal_str(s: &str) -> Result<BigRational> {
```

Comment:
Because we use `BigRational` everywhere. That would require implementing something like `rust_decimal_to_bigrational`. Not a big deal, but the `rust_decimal` lib itself will be used just to convert a string to it. It is a question of implementing 1 simple function vs. bringing additional dependency.

Reply:
Ah, nice. BigDecimal works fine. Thanks!